devito/passes/clusters/aliases.py

```diff
@@ -433,6 +441,9 @@ def rebuild(cluster, others, aliases, subs):
     return cluster.rebuild(exprs=exprs, ispace=ispace, dspace=dspace)
 
 
+# Utilities
+
+
```

Comment:
spaces?

Reply:
was like that before, I don't mind them to be honest, it denotes a different section of the file